content/en/docs/concepts/cluster-administration/node-shutdown.md

```diff
@@ -265,6 +265,27 @@ via the [Non-Graceful Node Shutdown](#non-graceful-node-shutdown) procedure ment
 {{< /note >}}
 
 
+## Windows Graceful node shutdown {#windows-graceful-node-shutdown}
+
+{{< feature-state feature_gate_name="WindowsGracefulNodeShutdown" >}}
+
+The Windows graceful node shutdown feature depends on kubelet running as a Windows service, 
+it will then have a registered [service control handler](https://learn.microsoft.com/en-us/windows/win32/services/service-control-handler-function) 
+to delay the  presshutdown event with a given duration.
+
+Windows graceful node shutdown is controlled with the `WindowsGracefulNodeShutdown` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) 
```

Comment:
```suggestion
Windows graceful node shutdown is controlled with the `WindowsGracefulNodeShutdown`
[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) 
```

Reply:
will  update, just wondering what the visual difference will be?

Reply:
From the perspective of the rendered HTML, nothing is different.
From the perspective of docs "source", we discourage long lines, because git diff works
on a line-by-line comparison basis. Changes to long lines are difficult to figure out
thus are difficult to review. For downstream localization teams, they need to track the
upstream changes on a line by line basis. Long lines make their lives suck.